Source/Core/Core/WiiRoot.cpp

```diff
@@ -143,12 +145,16 @@ void InitializeWiiFileSystemContents()
   SysConf sysconf{fs};
   sysconf.Save();
 
+  if (!NetPlay::IsNetPlayRunning())
+    ResetNetPlaySyncFS();
+
   InitializeDeterministicWiiSaves(fs.get());
 }
 
 void CleanUpWiiFileSystemContents()
 {
-  if (s_temp_wii_root.empty() || !SConfig::GetInstance().bEnableMemcardSdWriting)
+  if (s_temp_wii_root.empty() || !SConfig::GetInstance().bEnableMemcardSdWriting ||
+      s_netplay_sync_fs)
     return;
```

Comment:
This should be braced if the if condition (or body) travels across more than one line.